ansible_collections/arista/avd/examples/campus-fabric/README.md

```diff
@@ -0,0 +1,390 @@
+
+TASK [Copy all examples to ~/ansible-avd-examples]*****************************************************************************************************************************************************
+changed: [localhost]
+
+PLAY RECAP
+****************************************************************************************************************************************************************************************************************************************************************************
+localhost                  : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+```
+
+After the playbook has run successfully, the following directory structure will be created.
+
+``` shell
+ansible-avd-examples/     (directory where playbook was run)
+  ├── campus-fabric/
+    ├── documentation/
+    ├── group_vars/
+    ├── images/
+    ├── intended/
+    ├── switch-basic-configurations/
+    ├── ansible.cfg
+    ├── build.yml
+    ├── deploy.yml
+    ├── inventory.yml
+    └── README.md (this document)
+```
+
+???+ info
+
+    If the content of any file in the example is ***modified*** and the playbook is run again, the file ***will not*** be overwritten. However, if any file in the example is ***deleted*** and the playbook is run again, the file will be re-created.
+
+## Design Overview
+
+### Physical Campus Fabric Topology
+
+In a Campus network it is common to refer to the location of the switches as **MDF** (Main Distribution Frame) and **IDFs** (Independent Distribution Frame). Throughtout this example we refer to the spine nodes as the MDF and the leaf nodes as the IDFs. This example shows various switch types and common ways of cabling the IDF to the MDF.
+
+- MDF
+  - 2 Spine nodes
+- IDFs
+  - IDF1 supporting 192 users with a 2 leafs (2RU - 96 ports each)
+  - IDF2 supporting 240 users with a Modular 5 slot chassis (48 ports per module)
+  - IDF3 supporting 480 users with 5 leafs (2RU - 96 ports each)
+
+The drawing below shows the physical topology used in this example. The interface assignment shown here are referenced across the entire example, so keep that in mind if this example must be adapted to a different topology.
+
+![Figure: 1](images/campus_topo.svg)
+
+## Basic EOS Switch Configuration
+
+Basic connectivity between the Ansible controller host and the switches must be established before Ansible can be used to deploy configurations. The following should be configured on all switches:
+
+- Switch Hostname
+- IP enabled interface
+- Username and Password defined
+- Management eAPI enabled
+
+???+ info
+
+    When using vEOS/cEOS virtual switches, `Management0` is used. When using actual hardware switches, `Management1` is used. The included basic switch configurations may need to be adjusted for your environment.
```

Comment:
vEOS uses management1. To my knowledge only cEOS uses management0

Reply:
will reword this.  Not sure why cEOS and vEOS use diff management numbered interfaces...  Why not keep it the same...???

Reply:
Changed to:

The management interfaces used by cEOS and vEOS, are Management0 and Management1 respectively. When using actual hardware switches, Management1 is used. The included basic switch configurations may need to be adjusted for your environment.